app/ui/styles/components/spinner.css

```diff
@@ -46,6 +46,10 @@
   stroke: var(--content-default);
 }
 
+.spinner-secondary .path {
+  stroke: var(--content-secondary);
+}
+
```

Comment:
Decided against using this, but this fixes the `variant="secondary"` spinner to make the brighter part gray instead of green.

![image](https://github.com/user-attachments/assets/bccf4de3-dc32-4e33-9a80-876e44b1d19f)